internal/integration/influxdb/influxdb.go

```diff
@@ -195,6 +195,9 @@ func (i *Integration) HandleUplinkEvent(ctx context.Context, _ models.Integratio
 		Values: map[string]interface{}{
 			"value": 1,
 			"f_cnt": pl.FCnt,
+            "adr": pl.ADR,
```

Comment:
I think it would be better to move this to a tag value, as tags are indexed and values not. This would make aggregation and filtering more efficient.